src/lib/components/FieldCard.svelte

```diff
@@ -92,9 +92,10 @@
 </script>
 
 <div class="card rounded-none">
-  <figure on:click={() => goto(`/fields/${encodeURIComponent(field.id)}`)}>
+    <figure on:click={()=>goto(`/fields/${encodeURIComponent(field.id)}`)}>
```

Comment:
Looks like you need to get prettier into your editor or run it on everything before committing.